### DISCUSSION TITLE
How to make variometer Cat-01  in my TX16S to make emulated vSpd work?

### DISCUSSION THREAD

**Discussion by sukhoi222** [EdgeTX/edgetx#3113] [General]
I see from 2.7 version the problem with emulation of VSpd for variometre Cat-01 was solved with TX NV14. My question is how could I use this improvement to use vario Cat-01 in my TX16S(with Edge 2.8 installed and working) to make emulated vSpd work? 
Marc
Thank you

**Comment by kitsen13** [comment]
LUA?
